test/e2e/benchmarks.test-suite/memtier_benchmark/n4c16/test02-multi-memtier/code.var.sh

```diff
@@ -0,0 +1,63 @@
+# Redis parameters
+REDIS_PASS=abc123xyz
+
+test_time_minutes=3
+delay_between_rounds=30s
+memtier_threads=1
+test_time_seconds=$((test_time_minutes * 60))
+test_timeout_minutes=$((test_time_minutes + 2))
+
+# BM_* are benchmarks
+BM_MEMTIER="memtier-benchmark --server=redis-service --authenticate=$REDIS_PASS --threads=${memtier_threads} --test-time=${test_time_seconds}"
+
+# Clean up
+vm-command "kubectl delete jobs --all --now; kubectl delete deployment redis; kubectl delete service redis-service; kubectl delete secret redis; kubectl delete pods --all --now; true"
+
+vm-command "systemctl restart dbus"
+vm-command "systemctl restart kubelet"
+
+# Setup Redis
+wait="" create redis-secret
+CPU=4 MEM=8G CPULIM=8 MEMLIM=12G NAME=redis wait="Available" create redis
+NAME=redis-service wait="" create redis-service
+
+# Reset counters in order to keep creating pod0...
+reset counters
+
+benchmark_output_dir="$OUTPUT_DIR/benchmark/multi-memtier"
+mkdir -p "$benchmark_output_dir"
+
+export NAME=memtier-benchmark
+set +x
```

Comment:
nitpick: is this deliberately here, or is it a forgotten leftover from a debugging session ?

Reply:
@okartau Do we leave that `set +x` in place ?